### PR TITLE
README: Separate description about old version into another file

### PR DESCRIPTION
**Pull request by daipom**
It will be easier to understand because old version information is not necessary for many people.

### CODE REVIEW

**Comment by daipom** [comment]
I made minor improvements.

File: README.md [hunk 1]
```diff
@@ -9,128 +9,11 @@
 ## Installation
```
Comment (ashie) [inline README.md:9]: Could you add summary & caution for 2 plugins to head of this document?

e.g.)
```
This repository contains 2 Fluentd plugins:

* in_windows_eventlog
* in_windows_eventlog2

The former one is obsolete, please don't use in newly deployment.
This document describes about the later one. If you want to know about the obsolete one, please see [in_windows_eventlog(old).md](in_windows_eventlog(old).md)
```

Comment (daipom) [reply README.md:19]: I see! I added it!

**Review by ashie — APPROVED** [review]
LGTM

**Comment by ashie** [comment]
Thanks!

**Comment by daipom** [comment]
Thanks for your review!

**Pull request merged**